src/main/java/gregtech/integration/theoneprobe/provider/RecipeLogicInfoProvider.java

```diff
@@ -40,23 +42,26 @@ protected void addProbeInfo(@Nonnull AbstractRecipeLogic capability, @Nonnull IP
             }
             int EUt = capability.getInfoProviderEUt();
             int absEUt = Math.abs(EUt);
+            boolean consumer = capability.consumesEnergy();
             String text = null;
 
             if (tileEntity instanceof IGregTechTileEntity) {
                 IGregTechTileEntity gtTileEntity = (IGregTechTileEntity) tileEntity;
                 MetaTileEntity mte = gtTileEntity.getMetaTileEntity();
-                if (mte instanceof SteamMetaTileEntity) {
-                    text = TextFormatting.RED.toString() + absEUt + TextStyleClass.INFO + " L/t " + "{*material.steam*}";
+                if (mte instanceof SteamMetaTileEntity || mte instanceof MetaTileEntityLargeBoiler) {
+                    text = TextFormatting.RED.toString() + absEUt + TextStyleClass.INFO + " L/t {*gregtech.top.steam*} " + Materials.Steam.getLocalizedName();
```

Comment:
You cannot call `getLocalizedName()` here since this code is called server side, this won't properly translate in some cases. Instead do something like `"{*" + Materials.Steam.getUnlocalizedName() + "*}"`